File: app/src/main/java/com/yao/zhihudaily/model/ThemeJson.java
package com.yao.zhihudaily.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Administrator on 2016/9/10.
 * Daily详情
 */
public class ThemeJson {

    private ArrayList<ThemeStory> stories;
    private String description;
    private String background;
    private int color;
    private String name;
    private String image;
    private ArrayList<Editor> editors;
    @SerializedName("image_source")
    private String imageSource;

    public ArrayList<ThemeStory> getStories() {
        return stories;
    }

    public void setStories(ArrayList<ThemeStory> stories) {
        this.stories = stories;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getBackground() {
        return background;
    }

    public void setBackground(String background) {
        this.background = background;
    }

    public int getColor() {
        return color;
    }

    public void setColor(int color) {
        this.color = color;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public ArrayList<Editor> getEditors() {
        return editors;
    }

    public void setEditors(ArrayList<Editor> editors) {
        this.editors = editors;
    }

    public String getImageSource() {
        return imageSource;
    }

    public void setImageSource(String imageSource) {
        this.imageSource = imageSource;
    }

    @Override
    public String toString() {
        return "ThemeJson{" +
                "stories=" + stories +
                ", description='" + description + '\'' +
                ", background='" + background + '\'' +
                ", color=" + color +
                ", name='" + name + '\'' +
                ", image='" + image + '\'' +
                ", editors=" + editors +
                ", imageSource='" + imageSource + '\'' +
                '}';
    }
}

File: app/src/main/java/com/yao/zhihudaily/model/ThemeJson.kt
package com.yao.zhihudaily.model

import com.google.gson.annotations.SerializedName
import java.util.*

/**
 * @author Yao
 * @date 2016/9/10
 * Daily详情
 */
class ThemeJson {

    var stories: ArrayList<ThemeStory>? = null
    var description: String? = null
    var background: String? = null
    var color: Int = 0
    var name: String? = null
    var image: String? = null
    var editors: ArrayList<Editor>? = null
    @SerializedName("image_source")
    var imageSource: String? = null

    override fun toString(): String {
        return "ThemeJson{" +
                "stories=" + stories +
                ", description='" + description + '\''.toString() +
                ", background='" + background + '\''.toString() +
                ", color=" + color +
                ", name='" + name + '\''.toString() +
                ", image='" + image + '\''.toString() +
                ", editors=" + editors +
                ", imageSource='" + imageSource + '\''.toString() +
                '}'.toString()
    }
}
